algorithms/src/snark/marlin/marlin.rs

```diff
@@ -672,20 +717,10 @@ where
                     .iter()
                     .enumerate()
                     .flat_map(|(j, w_comm)| {
-                        [
-                            LabeledCommitment::new_with_info(
-                                &first_round_info[&witness_label(circuit_id, "w", j)],
-                                w_comm.w,
-                            ),
-                            LabeledCommitment::new_with_info(
-                                &first_round_info[&witness_label(circuit_id, "z_a", j)],
-                                w_comm.z_a,
-                            ),
-                            LabeledCommitment::new_with_info(
-                                &first_round_info[&witness_label(circuit_id, "z_b", j)],
-                                w_comm.z_b,
-                            ),
-                        ]
+                        [LabeledCommitment::new_with_info(
+                            &first_round_info[&witness_label(circuit_id, "w", j)],
+                            w_comm.w,
+                        )]
                     })
                     .collect_vec();
```

Comment:
I'm wondering if this call could be dropped

Reply:
febb1da7e